templates/flows/flow_results.haml

```diff
@@ -648,6 +652,34 @@
 
           %tbody{id:"results_run_table"}
             %tr{ic-append-from:"/flow/run_table/{{object.id}}", ic-trigger-on:"load", ic-target:"#results_run_table", ic-indicator:"#indicator"}
+        
+    #links.tab-pane
+      - if trackable_links
+        %table
+            %tr
+              %td.header
+                .details
+                  .text
+                    -trans "Contact"
+              %td.header
+                .details
+                  .text
+                    -trans "Link"
+              %td.header
+
+            %tbody{id:"results_run_table"}
```

Comment:
This ID should be different from `results_run_table` as we already have this one the runs table

Reply:
Renamed to `trackable_links_table`